src/config/index.js

```diff
@@ -100,8 +100,8 @@ const getAPIs = () => {
 };
 
 const getCookieSecret = () => {
-  if (_userSettings && _userSettings.cookieSecret) {
-    _cookieSecret = _userSettings.cookieSecret;
+  if (!_cookieSecret) {
+    throw new Error('COOKIE_SECRET is not defined in the environment variables.');
```

Comment:
Wouldn't this never trigger because of the fallback to `'cookie secret'`?

Reply:
Yeah sorry about that, I need to correct it.